testing/containers/ceph/Dockerfile

```diff
@@ -4,6 +4,8 @@ FROM ceph/daemon-base:latest-${CEPH_VERSION:-nautilus}
 ENV CEPH_VERSION=${CEPH_VERSION:-nautilus}
 
 RUN true && \
+    yum clean all && \
+    ( if [ "${CEPH_VERSION}" = "octopus" ]; then dnf install -y --repo BaseOS 'dnf-command(config-manager)' && dnf config-manager --set-disabled epel && dnf update -y; fi ) && \
     yum install -y git wget curl libcephfs-devel librados-devel librbd-devel /usr/bin/cc /usr/bin/c++ make && \
     (yum install -y /usr/bin/castxml || true) && \
```

Comment:
Does `castxml` not come from EPEL? The repository is now disabled, so it will never get installed.

You could do `yum --disablerepo="epel*" -y update` (using `epel*` to match all epel repositories) instead of the disabling the repository for all operations.

Reply:
That's correct. However, the epel install of castxml was already broken (for a week or so IIRC) and I didn't think it was important enough to block all CI over. I do intend for this to be a temporary situation, I just forgot to mention that aspect of it.

Oddly, I did try your --disablerepo suggestion and either I did something subtly different or wrong, but it didn't work for me.